### DISCUSSION TITLE
Github Hosted Runners Beta - Status

### DISCUSSION THREAD

**Discussion by samvantran** [community/community#32308] [Actions]
Hello from the GitHub-hosted Runners team 👋 

We’ll be reporting any incidents or issues with the service availability for the Large Runner’s beta here. This is separate from what we report on [GitHub Status](https://www.githubstatus.com/) which is for any incidents that impacts the overall Actions service and non-beta experiences.

If you notice any issues, you can check out this Discussion to see if there are any updates ❤️

**Comment by gdryke** [comment]
We are currently experiencing issues with the Larger Hosted Runners beta. We're working to restore service as quickly as possible, and will provide updates here as they are available.

**Reply by gdryke** [reply]
The issues are mitigated on our end and things should be back to normal.

**Comment by ritchxu** [comment]
Larger runners has graduated to GA, we'll be using https://www.githubstatus.com/ to communicate the service's status. Thanks!
